.drone/drone.jsonnet

```
@@ -92,6 +98,33 @@ local docker_manifest(app) = {
   },
 };
 
+local deploy_to_dev() = {
+  image: 'us.gcr.io/kubernetes-dev/drone/plugins/updater',
+  settings: {
+    config_json: |||
+        {
+          "destination_branch": "master",
+          "pull_request_branch_prefix": "cd-tempo-dev",
+          "pull_request_enabled": false,
+          "pull_request_team_reviewers": [
+            "tempo-squad"
```

Most teams use `-squad`, but our team is just `tempo` 🤷🏻 
https://github.com/orgs/grafana/teams/tempo

```suggestion
            "tempo"
```

---

Aha :)